tools/internal_ci/helper_scripts/prepare_build_linux_perf_rc

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+# Copyright 2017 gRPC authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Source this rc script to prepare the environment for linux perf builds
+
+# Need to increase open files limit and size for perf test
+ulimit -n 32768
+ulimit -c unlimited
```

Comment:
The sync tests need as many threads as they have RPCs outstanding, plus some additional threads for executors and overhead type of things. Most of the sync tests run with 1000 outstanding RPCs, so I think the default value (of 32K or something) should be plenty even if you have multiple clients and servers on the same machine.